cassandra-release/prepare_release.sh

```diff
@@ -326,21 +332,28 @@ then
     execute "cd $rpm_package_dir"
 
     echo "Building redhat packages ..." 1>&3 2>&4
+    execute "mkdir $tmp_dir/cassandra-dist-dev/${release}/redhat"
```

Comment:
We can move `${tmp_dir}/cassandra-dist-dev/${release}/redhat/` to its own dedicated variable as it appears 7 times in the code below.

Reply:
i don't want to touch this file, except for when i'm actually cutting a release. so i'm keen to call it out-of-scope for this work for now (despite it being in the patch).

Reply:
Make note to create a follow up JIRA for it perhaps?